imports/ui/pages/signup/students.js

```diff
@@ -10,10 +10,12 @@ var amountLanguages = [];
 var amountExperience = [];
 
 Template.studentSignup.onRendered(function(){
+  var template = this;
+  var link = this.findAll("a");
```

Comment:
It was adding disabled to the login form input button, because the click event was also being triggered when we clicked in the **a** tag in the top left:
![screenshot_1](https://cloud.githubusercontent.com/assets/4267241/19078300/0f32bb60-8a26-11e6-9eb5-654bf7ce4a80.png)

which goes back to the home. Now it only triggers if we click in an **a** tag inside the template **studentsSignup**